tests/test_driver.py

```diff
@@ -261,34 +248,28 @@ def run_test(
         )
         if not PEXPECT:
             return TestSkip(arg)
-        try:
-            proc = subprocess.run(
-                ["python3", path],
-                capture_output=True,
-                env=pyenviron,
-                # Timeout of 120 seconds, about 10 times what any of these takes
-                timeout=120,
```

Comment:
yeah I don't expect the timeout was needed because pexpect already has its own timeouts